### ISSUE TITLE
The new app "Crypto Alerts" - alternative to the Bitcoin Checker.

### ISSUE THREAD

**Issue by Streamatico** [aneonex/BitcoinChecker#241]
Try the new app "[Crypto Alerts](https://play.google.com/store/apps/details?id=com.streamatico.cryptoalert)" - it's an alternative to the old Bitcoin Checker.

I have made an effort to implement the features of the old app and add new functionalities, including:

- Modern interface
- Charts for most popular exchanges
- Automatic update of the currency pairs list
- and many other minor features

The app is completely free to use with up to 5 cryptocurrency pairs.
For more than 5 pairs, a paid subscription is required, with a minimal price to cover development costs and support further improvements.

I hope you will enjoy using this app.

Install from Google Play: https://play.google.com/store/apps/details?id=com.streamatico.cryptoalert

**Comment by bryankaplan** [comment]
Thank you! The UI looks pleasantly familiar. But is it open source?

**Comment by Streamatico** [comment]
No, the source code is closed. It is based on open-source [data module](https://github.com/aneonex/BitcoinChecker) that is distributed under the MIT license. If necessary, I may open a redesigned data module used in the new application in the future. I don’t plan to fully open the app code yet.

**Comment by aCID-sLAM** [comment]
I use no google service, so no way to unlock more than 5 pairs for me. I stay with the old version and cross my fingers.

**Comment by Cypher01** [comment]
Wtf, seriously?? This is obviously a clone of the old free open source projects. You definitely didn't build it from scratch, the code base is stolen. And you have the impudence to close the source code and charge money?!?

I call for a boycott! Everyone, don't use the app, but report it!

**Comment by Streamatico** [comment]
The code is completely new. So that there is no doubt that this new app, I will publish most of the app into Open Source.

P.S.
I myself used the old Bitcoin Checker and I was sad to see that it disappeared from Google Play and was not developed, this was the impetus for creating an alternative. I spent a lot of time and effort implementing the app.

**Comment by aCID-sLAM** [comment]
Maybe you consider a alternative way for unlocking more than 5 exchanges, without google play.

**Comment by Streamatico** [comment]
@aCID-sLAM Ok, I'll think about it. But I still don’t understand how to update the app without Google Play. How do you currently install and update other apps? Downloading APK manually is an inconvenient and unreliable method.

**Comment by aCID-sLAM** [comment]
https://gitlab.com/AuroraOSS/AuroraStore

**Comment by Satnow79** [comment]
It's not available on Huwaei's AppGallery!

Is it compatible with Huwaei EMUI 13?

**Comment by KoolZero** [comment]
Is a damn fu*king bad copy! 😡

**Comment by donjoe0** [comment]
@aCID-sLAM 
I just tried it via Aurora on a degoogled Android 13 and it failed to install.

@Streamatico 
Can you please publish APKs in the Resources of your GitHub anyway, for those of us masochistic enough to want to update manually? 🙂 You could restrict that to major updates only, if it's too much work to put every little thing on there, and it would still be better than not having it at all on degoogled phones.

Of course it would be much easier if you open-sourced it because then you could publish it on FOSS app stores like Accrescent or F-Droid, but... yeah.
LE: Oh wait, Accrescent allows closed-source apps!

@KoolZero 
Why? What's missing that was important to you?
